src/portfolio-rebuild/styles.css

```diff
@@ -0,0 +1,230 @@
+
+body {
+  width: 100%;
+  height: auto;
+  border: 2px solid black;
+  padding: 0px;
+  margin: 0px;
+}
+
+.top-nav {
+  display: flex;
+  width: 100%;
+  height: 10%;
+  background-color: hotpink;
+  justify-content: space-between;
+  box-shadow: 2px 2px 10px;
```

Comment:
add a color! and play with the opacity: `box-shadow: 2px 2px 10px rgba(0, 0, 0, 0.26);` . remember you can open up the console, click on top nav, click on the box-shadow and play with it there!